{"name": "test_404_error", "status": "passed", "description": "Тест обработки ошибки 404", "attachments": [{"name": "stdout", "source": "d99b54ed-0e98-4b26-accf-a96d7e733cc0-attachment.txt", "type": "text/plain"}], "start": 1761097055811, "stop": 1761097056839, "uuid": "498ae779-cf35-467c-b9e5-3ddaeff3719c", "historyId": "8c5edd45f53b33f4e3adabdf11981ca0", "testCaseId": "8c5edd45f53b33f4e3adabdf11981ca0", "fullName": "tests.test_api.TestJSONPlaceholderAPI#test_404_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestJSONPlaceholderAPI"}, {"name": "host", "value": "Dimon"}, {"name": "thread", "value": "10628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestJSONPlaceholderAPI"]}